openmetadata-ui/src/main/resources/ui/src/components/Domain/DomainPage.component.tsx

```diff
@@ -189,41 +186,43 @@ const DomainPage = () => {
     );
   }
 
-  if (isEmpty(rootDomains)) {
-    return (
-      <ErrorPlaceHolder
-        buttonId="add-domain"
-        className="mt-0-important"
-        heading={t('label.domain')}
-        permission={createDomainPermission}
-        type={
-          createDomainPermission
-            ? ERROR_PLACEHOLDER_TYPE.CREATE
-            : ERROR_PLACEHOLDER_TYPE.CUSTOM
-        }
-        onClick={handleAddDomainClick}>
-        {t('message.domains-not-configured')}
-      </ErrorPlaceHolder>
-    );
-  }
-
   return (
-    <ResizableLeftPanels
-      className="content-height-with-resizable-panel"
-      firstPanel={{
-        className: 'content-resizable-panel-container',
-        minWidth: 280,
-        flex: 0.13,
-        children: <DomainsLeftPanel domains={rootDomains} />,
-      }}
-      pageTitle={t('label.domain')}
-      secondPanel={{
-        children: domainPageRender,
-        className: 'content-resizable-panel-container p-t-sm',
-        minWidth: 800,
-        flex: 0.87,
-      }}
-    />
+    <PageLayoutV1 pageTitle={t('label.domain-plural')}>
+      {isEmpty(rootDomains) ? (
+        <ErrorPlaceHolder
+          buttonId="add-domain"
+          className="mt-44"
+          heading={t('label.domain')}
+          permission={createDomainPermission}
+          type={
+            createDomainPermission
+              ? ERROR_PLACEHOLDER_TYPE.CREATE
+              : ERROR_PLACEHOLDER_TYPE.CUSTOM
+          }
+          onClick={handleAddDomainClick}>
+          {t('message.domains-not-configured')}
+        </ErrorPlaceHolder>
+      ) : (
+        <ResizableLeftPanels
+          className="content-height-with-resizable-panel"
+          firstPanel={{
+            className: 'content-resizable-panel-container',
+            minWidth: 280,
+            flex: 0.13,
+            children: <DomainsLeftPanel domains={rootDomains} />,
+          }}
+          pageTitle={t('label.domain')}
+          secondPanel={{
+            children: domainPageRender,
+            className: 'content-resizable-panel-container p-t-sm',
+            minWidth: 800,
+            flex: 0.87,
+          }}
+        />
+      )}
+
+      {domainLoading && <Loader />}
```

Comment:
Isn't here we will show loader and content conditionally? 
like: 
```
domainLoading ? <Loader /> : Conten?
```

Reply:
Yeah. we can.